lrlex/src/lib/parser.rs

```diff
@@ -362,6 +372,15 @@ impl<StorageT: TryFrom<usize>> LexParser<StorageT> {
         Ok(i + line_len)
     }
 
+    fn parse_start_state_ops<'a>(&self, start_state_str: &'a str) -> (&'a str, StartStateOperation) {
+        let (left_delta, operation) = match start_state_str.chars().nth(0).unwrap_or_default() {
```

Comment:
Seems, I lost one comment in my review, 

`cargo clippy --all-targets` is going to want `nth(0)` -> `next()`, here and in the tests

Reply:
Done, in 6da39479cf110d8bc9ffd9dea3a2bcf0badc8f8d